### PR TITLE
fixing and updating docs

### PR DESCRIPTION
**Pull request by bardkw**

### CODE REVIEW

File: docs/release_notes/pending_release.rst [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
+This is the initial release of ``smqtk-indexing``, spinning off from
+v0.14.0 of the monolithic `SMQTK`_ library.
+
+.. _smqtk: https://github.com/kitware/smqtk
 
 Updates / New Features
 ----------------------
 
+Misc.
+* Fixing documentation and adding RTD
```
Comment (Purg) [inline docs/release_notes/pending_release.rst]: I believe reST will complain about there not being whitespace between a bullet and the previous line.
```suggestion
Misc.

* Fixing documentation and adding RTD
```

File: docs/relevancyinterfaces.rst [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+for the pool vectors and additionally returns a sequence of UIDs,
+possibly not all from the pool, on which feedback would be most
+useful.
+
+.. autoclass:: smqtk_relevancy.interfaces.rank_relevancy.RankRelevancyWithFeedback
+   :members:
+   :private-members:
+
+
+RelevancyIndex
```
Comment (Purg) [inline docs/relevancyinterfaces.rst:36]: I'm debating if we should add a deprecation notice here and refer to usage of `RankRelevancy`.

File: docs/relevancyimplementations.rst [hunk 3]
```diff
@@ -0,0 +1,20 @@
+Relevancy Implementations
+-------------------------
+
+RankRelevancyWithMarginSampledFeedback
+++++++++++++++++++++++++++++++++++++++
+
+.. autoclass:: smqtk_relevancy.impls.margin_sampling.RankRelevancyWithMarginSampledFeedback
```
Comment (Purg) [inline docs/relevancyimplementations.rst]: This and the other two autodocs are failing noting that the module path provided does not resolve. I think you are missing a module after `impls`.

File: docs/sphinx_server.py [hunk 4]
```diff
@@ -15,14 +15,13 @@
 from livereload import Server, shell
 
 rebuild_cmd = shell('make html', cwd='.')
+rebuild_root = "_build/html"
+watch_source_dir = "../smqtk_relevancy"
 
 watch_dirs = [
     '.',
-    'architecture',
     'examples', 'examples/nnss_incremental_update',
```
Comment (Purg) [inline docs/sphinx_server.py]: These don't exist in this repo.

File: docs/sphinx_server.py [hunk 5]
```diff
@@ -15,14 +15,13 @@
 from livereload import Server, shell
 
 rebuild_cmd = shell('make html', cwd='.')
+rebuild_root = "_build/html"
+watch_source_dir = "../smqtk_relevancy"
```
Comment (Purg) [inline docs/sphinx_server.py:19]: This is not being used below. L38 is still hard-coded to refer to an old directory from the mono-repo.

**Review by Purg — CHANGES_REQUESTED** [review]

File: docs/relevancyinterfaces.rst (hunk 2 above)
Comment (bardkw) [reply docs/relevancyinterfaces.rst:36]: what would be the correct way to signal this here?
simple adding a `[Deprecated]` under the header and a note to refer to use `RankRelevancy` instead?

Comment (Purg) [reply docs/relevancyinterfaces.rst:36]: I don't know a canonical "best" way, but I imagine we could add a narrative sentence as the first thing under this section so it is blatantly obvious (like make it bold or something).

**Comment by bardkw** [comment]
squashed commits and resolved outstanding issues

**Review by Purg — APPROVED** [review]

**Pull request merged**